index.js

```diff
@@ -204,6 +188,12 @@ function testGenResponse(swagger, path, operation, response, config) {
 
   // get the data
   data = getData(swagger, path, operation, response, config);
+  if (produce === 'application/json' && !data.noSchema) {
```

Comment:
I use ```const``` and get ```SyntaxError: Use of const in strict mode```

Issue addressed [here](http://stackoverflow.com/questions/22603078/syntaxerror-use-of-const-in-strict-mode)

shall i just use ```var``` then?